draft-irtf-cfrg-opaque.md

```diff
@@ -551,15 +516,8 @@ The authenticated key exchange flow is shown below:
 
 These messages are named `KE1`, `KE2`, and `KE3`, respectively. They carry the
 messages of the concurrent execution of the key recovery process (OPRF) and the
-authenticated key exchange (AKE):
-
-- `KE1` is composed of the `CredentialRequest` and `AuthInit` messages
-- `KE2` is composed of the `CredentialResponse` and `AuthResponse` messages
-- `KE3` represents the `AuthFinish` message
-
-The `CredentialRequest` and `CredentialResponse` message contents and wire
-format are specified in {{cred-retrieval}}, and those of `AuthInit`,
-`AuthResponse` and `AuthFinish` are specified in {{ake-messages}}.
```

Comment:
Why remove this?

Reply:
It doesn't seem to add any value (to me, after not having looked at it for quite some time).